source/tutorials/learning-paths/pipelines/1-get-started-pl.rst

```diff
@@ -0,0 +1,156 @@
+.. meta::
+    :description: Valohai Pipelines learning path - Creating your first pipeline in Valohai
+
+Getting started
+#################
+
+.. include:: _intro-pl.rst
+
+In this section you will learn:
+
+- What is a pipeline?
+- What are pipeline nodes?
+- What are pipeline edges?
+
+The sample scripts used in this tutorial can also be found here. 
+
+
+What is a pipeline?
+--------------------
+
+Pipelines automate your machine learning operations on Valohai ecosystem. They are series of executions, Tasks and deployments.
+
```

Comment:
I'd give here a few examples of pipelines:

Something like 

* A pipeline that runs every Monday at 09:00 to fetch data from a database, and if there are over 1000 new records it will retrain the model, evaluate its performance, and update the `latest-model` alias to the newly trained model.
* A pipeline that trains 300 models in parallel and compares if any of those models is better that the existing model
* A pipeline that trains a new model and deploys a new version of our REST API with the new model.